types/altv-client.d.ts

```diff
@@ -463,6 +469,11 @@ declare module "alt-client" {
   }
 
   export class Discord {
+    /*
+     * Returns the current Discord user. Can be null.
+     *
+     * @remarks currentUser is only available if and only if Discord and alt:V are both started with administrator priviledges.
```

Comment:
It should be described as something like `Requires both alt:V and Discord to be run with administrator privileges.`